src/core/components/start.js

```diff
@@ -45,7 +45,9 @@ module.exports = (self) => {
       self._bitswap = new Bitswap(
         self._libp2pNode,
         self._repo.blocks,
-        self._peerInfoBook
+        Object.assign({
+          statsEnabled: true
+        }, self._peerInfoBook)
```

Comment:
This is weird. Are you saying that in order to enable stats on Bitswap there is a variable on PeerBook that needs to be true? This Object Assign is also probably messing with the fact that PeerBook is a Object from Class and not an Object Literal. //cc @pgte

Reply:
We need to have the stats enabled and this was the best way I found to do that.

Reply:
@pgte could you comment here given that you implemented stats on Bitswap

Reply:
I see. the thing is that bitswap doesn't need access to the PeerInfoBook at all. You can just pass the object options.